keyboards/handwired/split89/info.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
This file does not match the defined layout, so is producing the following `qmk info` output (and would also incorrectly render in configurator)
```
Keyboard Name: split89
Manufacturer: jurassic73
Website: 
Maintainer: jurassic73
Keyboard Folder: handwired/split89
Layouts: LAYOUT
Size: 3 x 2
Processor: atmega32u4
Bootloader: atmel-dfu
Layout:
┌──┐┌──┐┌──┐        ┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐┌──┐
│k0││k0││k0│        │k0││k0││k0││k0││k0││k1││k1││k1││k1││k1││k1││k1││k1││k1││k1││k2││k2││k2││k2││k2││k2││k2││k2││k2││k2││k3││k3││k3││k3││k3││k3││k3││k3││k3││k3││k4││k4││k4││k4││k4││k4││k4││k4││k4││k4││k5││k5││k5││k5││k5││k5││k5││k5││k5││k5││k6││k6││k6││k6││k6││k6││k6││k6││k6││k6││k7││k7││k7││k7││k7││k7││k7││k7││k7││k7││k8││k8││k8││k8││k8││k8││k8││k8││k8│
└──┘└──┘└──┘        └──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘└──┘
┌────┐┌────┐
│k10 ││k12 │
└────┘└────┘
```

Reply:
Thanks for this.  I will get this sorted.

Reply:
This has been corrected.  Thanks!